telematic_system/telematic_cloud_messaging/Dockerfile

```diff
@@ -3,6 +3,7 @@ WORKDIR /telematic_cloud_messaging
 COPY ./ /telematic_cloud_messaging/
 RUN ./mvnw clean package -Dmaven.test.skip=true
 RUN mv /telematic_cloud_messaging/target/*.jar app.jar
+RUN mv /telematic_cloud_messaging/src/main/resources/application.properties /telematic_cloud_messaging/application.properties
```

Comment:
Why does the application.properties file need to be moved here?

Reply:
so that the application can read this properties file